SpatialGDK/Documentation/content/how-to-use-schema.md

```diff
@@ -6,45 +6,57 @@
 
-    Select **Schema** in the [GDK Toolbar]({{urlRoot}}/content/toolbars#buttons). The GDK automatically iterates through classes* that are currently loaded by the Editor, generates the schema files and updates the [SchemaDatabase]({{urlRoot}}/content/glossary#schemadatabase).<br/>
+    Select **Schema** in the [GDK toolbar]({{urlRoot}}/content/toolbars#buttons). The GDK automatically iterates through classes that are currently loaded by the Editor, generates the schema files and updates the [SchemaDatabase]({{urlRoot}}/content/glossary#schemadatabase).<br/>
     ![Toolbar]({{assetRoot}}assets/screen-grabs/toolbar/schema-button.png)<br/>
-    _Image: In the GDK toolbar in the Unreal Editor, select **Schema**_<br/><br/>
+    _Image: In the GDK toolbar in the Unreal Editor, select **Schema**_<br/>
 
-As the GDK automatically generates all the schema you need, you do not have to write or edit schema manually when using the GDK.
+### Generating schema via the command line
+You can generate schema via the command line using an Unreal commandlet. This method uses a Full Scan (see [Which method to use](#which-method-to-use)). It's useful as part of an automated build system.
 
-> \*Whenever you generate schema, the GDK automatically creates schema for classes with [replicated properties (Unreal documentation)](https://docs.unrealengine.com/en-US/Gameplay/Networking/Actors/Properties) or [RPCs (Unreal documentation)](https://docs.unrealengine.com/en-us/Gameplay/Networking/Actors/RPCs). If a class does not have replicated properties or RPCs, then the GDK does not generate schema for it. 
+To generate schema via the command line, run the following command:
 
-## When to generate schema
+```
+<Path to UE4Editor.exe> <Path to your game's .uproject file> -run=GenerateSchemaAndSnapshots -MapPaths=<MapName1;MapName2;MapName3>
+```
 
-Select **Schema (Full Scan)** if you have: 
+This command generates schema using a Full Scan for all assets, and generates individual snapshots (`<MapName>.snapshot`) for each map. Any specified map paths that end in `/` are interpreted as a directory, and snapshots are generated for each .umap found under these paths.
 
-* Not generated any schema for your project.
-* Added a class with replicated properties that is not currently loaded by the Editor.
-* Edited a class with replicated properties that is not currently loaded by the Editor.
+As the GDK automatically generates all the schema you need, you do not have to write or edit schema manually when using the GDK.
 
-Select **Schema** if you have:
+## Which method to use
 
-* Added a class with replicated properties that is currently loaded by the Editor.
-* Edited a class with replicated properties that is currently loaded by the Editor.
+Select **Schema (Full Scan)** or generate schema via the command line if you have: 
+
+* not generated any schema for your project.
+* added or edited a class with replicated properties that is not currently loaded by the Editor.
```

Comment:
Added or edited a class since I last generated Schema (Full Scan)?
Also, is it only a replicated properties or RPCs too?

Reply:
Good point - I'm guessing yes (to your first question), but am not sure about your second. The original doc only mentioned replicated properties, but that doesn't necessarily mean it's right! cc @padhansell

Reply:
Sorry yep this means both RPCs and properties. When talking about schema generation, it will always refer to both